wear/src/main/java/io/homeassistant/companion/android/util/CommonFunctions.kt

```diff
@@ -8,9 +8,13 @@ import com.mikepenz.iconics.IconicsDrawable
 import com.mikepenz.iconics.typeface.IIcon
 import com.mikepenz.iconics.typeface.library.community.material.CommunityMaterial
 import io.homeassistant.companion.android.common.data.integration.Entity
+import io.homeassistant.companion.android.home.HomePresenterImpl
 import java.util.Calendar
 import io.homeassistant.companion.android.common.R as commonR
 
+fun stringForDomain(domain: String, context: Context): String? =
+    HomePresenterImpl.domainsWithNames[domain]?.let { context.getString(it) }

```

Comment:
This should probably be extended / adjusted because there are currently a lot of entities not supported by the app, which will now show up as something like "automation" or "device_tracker"

Reply:
True, but in the current implementation the keys of this list are taken as supported entities in `HomePresenter`, so maybe need to extend the existing list with all possible domains...

Reply:
By the way, the list of domains is basically endless, but I'll start with including the most common domains :)

Reply:
> in the current implementation the keys of this list are taken as supported entities in `HomePresenter`

Right, so I'd say you need to use another list for the strings or change the function that returns the supported domains 🙂

> By the way, the list of domains is basically endless

Good point. Perhaps a nicer fallback string would be a good alternative? (replace underscores with spaces, capitalize the string)

Reply:
> Good point. Perhaps a nicer fallback string would be a good alternative? (replace underscores with spaces, capitalize the string)

Good suggestion :)